config/config/sc-config.yaml

```
@@ -67,7 +67,7 @@ user:
       enabled: false
 
 harbor:
-  enabled: true
+  enabled: false
```

Harbor should still be enabled. But disable Harbor backups on line 154: https://github.com/elastisys/compliantkubernetes-apps/blob/25bd108d6832661522b4d84537eeecd11c109aca/config/config/sc-config.yaml#L154
```suggestion
  enabled: true
```